### ISSUE TITLE
photo frame geometry

### ISSUE THREAD

**Issue by GoogleCodeExporter** [yendo/gphotoframe#83]
```
photoframe version 1.2
ubuntu 10.10

Can you add an option to Lock the frame position? Every time I restart my 
computer, photoframe is at the top left corner, not in the position where I 
left it at.

Can you add a visibility option so that photoframe will not hide when I click 
on the 'Show Desktop' applet?  When I click 'Show Desktop', I want to see 
photoframe, but instead it gets hidden along with all open windows.

Also, when I initiate the Compiz scale window, photoframe should hide and not 
be shown along with the other scaled windows.

Thanks.

```

Original issue reported on code.google.com by `sicklem...@inbox.com` on 9 Feb 2011 at 5:04

**Comment by GoogleCodeExporter** [comment]
```
> Can you add an option to Lock the frame position? 
> Every time I restart my computer, photoframe is at the top left corner, not 
in the position where I left it at.

gphotoframe should save it's own geometry.
Could you show me the result of thefollowing command?

$ cat ~/.gconf/apps/gphotoframe/%gconf.xml 

> Can you add a visibility option so that photoframe will not hide when I click 
on the 'Show Desktop' applet?  
> When I click 'Show Desktop', I want to see photoframe, but instead it gets 
hidden along with all open windows.
> 
> Also, when I initiate the Compiz scale window, 
> photoframe should hide and not be shown along with the other scaled windows.

I opened new issue #84.

```

Original comment by `yendo0206` on 9 Feb 2011 at 5:22
- Changed title: **photo frame geometry**

**Comment by GoogleCodeExporter** [comment]
```
All of my issues were solved by enabling "Fix Photo Frame".  I was just 
confused by its purpose.  I thought "Fix Photo Frame" is used to fix (redraw) 
the frame if it doesn't appear correctly.  Wouldn't "Lock position" be a more 
suitable term to use? 
Thanks again.
```

Original comment by `sicklem...@inbox.com` on 9 Feb 2011 at 8:02

**Comment by GoogleCodeExporter** [comment]
```
"Fix Photo Frame" is for the issue #1 (issue #68).
The menu item does noting for saving the photo frame geometry.
```

Original comment by `yendo0206` on 11 Feb 2011 at 12:13
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
I tested photoframe by repeatedly logging out & logging in my desktop. 
With "Fix Photo Frame" disabled, photoframe 
  - always appear on top left corner
  - minimize to taskbar with Show Desktop
  - it scales along with other windows with Compiz scale command

With "Fix Photo Frame" enabled, photoframe
  - appears where I last placed it, moves a little bit
  - not affected by Show Desktop
  - doesn't scale with Compiz scale, stays below other scaled windows

$ cat ~/.gconf/apps/gphotoframe/%gconf.xml
<?xml version="1.0"?>
<gconf>
    <entry name="window_fix" mtime="1297453773" type="bool" value="true"/>
    <entry name="interval" mtime="1297127946" type="int" value="60"/>
    <entry name="max_height" mtime="1297127819" type="int" value="150"/>
    <entry name="max_width" mtime="1297127815" type="int" value="200"/>
    <entry name="root_y" mtime="1297453773" type="int" value="791"/>
    <entry name="root_x" mtime="1297453773" type="int" value="1783"/>
    <entry name="fullscreen" mtime="1297453150" type="bool" value="false"/>
</gconf>

Anyway, my issue is solved.  
```

Original comment by `sicklem...@inbox.com` on 11 Feb 2011 at 8:09
